index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <Title>Etchasketch</Title>
+    <link href="./style.css" type="text/css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Great+Vibes" rel="stylesheet">    
+</head>
+
+<body>
+<div class ='gameContainer' id='gameContainer'>
+    <div class='rightWheel'>
+    </div>
+    <div class='leftWheel'>
+    </div>
+    <button class='resetButton' id='resetButton'>Reset</button>
+    <div class='gameTitle'>
+        <h1>Magic Etch A Sketch</h1>
+    </div>
+</div>
+
+<script>
+
+document.getElementById('resetButton').addEventListener('click',() => { makeGrid()});
```

Comment:
JS code should be indented correctly

Reply:
Could you move the JS into an external file and then include it using a script tag please